sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/json/JsonTokenType.java

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.json;
+
+/**
+ * Enumeration of JSON token types.
+ */
+enum JsonTokenType {
+
+    /**
+     * Reserved for no other value.
+     */
+    NOTSTARTED,
```

Comment:
NIT - I would prefer either using CamelCase as in .Net or if you use all-caps separate words with a _

In general I think sticking to what you have in .Net is fine. This is an implementation package API - will never be in public surface area - so, we have some liberty on how to use naming conventions.

Reply:
Updated all enums to be PascalCase